src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -453,14 +480,23 @@ private static void init(){}
          */
         public static final TagKey<Item> TOOLS_FISHING_RODS = tag("tools/fishing_rods");
         /**
-         * A tag containing all existing tridents.
+         * A tag containing all existing shears.
          *
          * Note: This tag is not an alternative or a substitute to {@link net.minecraftforge.common.ToolActions}.
          *
          * @see net.minecraftforge.common.ToolAction
          * @see net.minecraftforge.common.ToolActions
          */
-        public static final TagKey<Item> TOOLS_TRIDENTS = tag("tools/tridents");
+        public static final TagKey<Item> TOOLS_SHEARS = tag("tools/shears");
+        /**
+         * A tag containing all existing tridents or spears. (Essentially throwable weapons)
+         *
+         * Note: This tag is not an alternative or a substitute to {@link net.minecraftforge.common.ToolActions}.
+         *
+         * @see net.minecraftforge.common.ToolAction
+         * @see net.minecraftforge.common.ToolActions
+         */
+        public static final TagKey<Item> TOOLS_SPEARS = tag("tools/spears");
```

Comment:
Is this for spears or throwable weapons? those are two very different things. Is a boomerang, a throwing knife, or a shuriken a "spear"? I would also call a quarterstaff a spear despite it not being throwable.

Reply:
This one is am having trouble with for the name but basically, if I name this tridents, it might prevent people from adding their throwable impaling stick weapon into here. So I broadened the name from trident to spears because tridents is a subset of spears (it's a 3 prong spear by definition)

Boomerangs and other throwing stuff aren't spears and don't belong in this tag as they aren't spears. Instead, modders for those should make their own tool tag for those kinds of weapons as someone querying spears to get a boomerang or throwing knife is going to be very confused.

What I can do I expand the javadoc to explicitly state it is for throwing spears like tridents and anything else is best made into their own tag.

Reply:
for spears, i would suggest `trident_like`. yes i agree that tridents are sub-type of spear, but mc game introduced tridents first so it makes sense to call the tag trident_like. similar modded weapons (spears, bidents) will be based on trident.

Reply:
I thought some more and yeah `spears` is totally fine. Example, if someone makes a Sword that is throwable like a Trident, it should still go in the `swords` tag as it is a sword, not a trident despite having the same behavior of a trident. These tags are classifying the kind of tool, not the behavior of these tools. 

Thus even non-throwable spears should be allowed here as well. I went ahead and removed the `throwable` part of the javadoc out so now it should be clear that this is about the tool itself and not the behavior.

(For some background, the tag was originally `c:spears` on Fabric so they have been running witht hat for a while and haven't been having any problems)

If that's good with y'all, I'll mark this convo as resolved